### DISCUSSION TITLE
ライフレートとステータスの最大値を100固定で開始するオプションが欲しい

### DISCUSSION THREAD

**Discussion by mikkatenka** [hengband/hengband#3625] [Ideas]
新しくゲームを始めて自己分析をしたとき
ライフレートと耐久の値が低いとプレイヤーの意欲が下がってしまう
新生でステータス厳選とかなんかつかれる…やりたくない…
そのような心境の時に100固定のオプションがあればやる気が湧いてくると思います
有利不利なく平均的なプレイ感が味わえるのも良い事かと
ぜひお願いしたいです

**Comment by Hourier** [comment]
ご意見ありがとうございます
技術的な回答としては「今のところ困難」の一点張りになってしまいます
無理すれば何とか作れそうには思いますが、5年後の誰か (下手すると自分)に無理のツケを払わせる結果が待っています
申し訳ありません
Discord ではユーザ含めて色々な意見が出ていますので、ぜひいらして頂けると助かります

**Comment by daradarach** [comment]
性格「ふつう」の時に限りライフレート100、全能力18/100にしてはどうかなんて意見があったのでメモ
